spec/support/features_support.rb

```diff
@@ -23,3 +23,23 @@ def expect_an_error(hash_pair = {})
   expect(page).to have_css(error_div)
   expect(find("#{error_div} span.error")).to have_content(msg)
 end
+
+# Assume regular user (member) is logged in
+#
+# assume_logged_in
```

Comment:
this is not commended code, it is comment with example how to use this helper methods